apps/glusterfs/operations_manage.go

```diff
@@ -14,60 +14,96 @@ import (
 	"net/http"
 	"sync"
 
+	"github.com/lpabon/godbc"
+
 	"github.com/heketi/heketi/executors"
+	"github.com/heketi/heketi/pkg/idgen"
+)
 
-	"github.com/lpabon/godbc"
+type OpClass int
+
+const (
+	TrackNormal OpClass = iota
+	TrackToken
 )
 
-// OpCounter is used to track and manage how many operations are being
+// OpTracker is used to track and manage how many operations are being
 // processed by the server.
-type OpCounter struct {
+type OpTracker struct {
 	// configuration
 	Limit uint64
```

Comment:
can this be converted to lower case?

Reply:
Possibly, but I'm not sure what the value of making it private would be. Do you see the limit value as an implementation detail that should not be exposed?